content/en/blog/2025/roadmap/index.md

```diff
@@ -0,0 +1,43 @@
+
+## 2025 Themes
+
+### Sidecar to ambient migration
+
+With the promotion of Istio ambient mode to Stable, the world’s fastest service mesh is also the world’s most efficient, and is now easier to operate than ever. With graduation, we’ve seen a substantial increase in interest, and a corresponding number of requests for a comprehensive migration guide for existing sidecar users.  While our previous efforts to stabilize ambient mode have been targeted at new Istio users, it is clear that the time has come to provide an onramp for our existing user base to migrate to ambient mesh.  While the technical foundations for this migration have been in place for some time (and some brave users have migrated on their own), we will be making new investments in automated migration testing, tooling to assess your readiness to migrate, and documentation to guide users every step of the way.
+
+In addition to tests, tooling, and documentation, users migrating between data planes should reasonably expect that the Istio features they know and love will continue to work in their new environment.  For this reason, we are investing in closing the most significant functionality gaps between sidecar and ambient mode, specifically by adding support for multi-cluster traffic management and extensibility, which you can read about below.
+
+As we have stated in previous years, we have no intention of ending support for sidecar mode as long as there are users for it.  Migrating to ambient mesh is completely voluntary, and we expect many users will use sidecars for years to come.
+
+### Multi-cluster ambient mesh
+
+Multi-cluster traffic management has long been one of the most valued enterprise features of Istio, and we are hard at work to bring this value to ambient mode users in 2025.  With a multi-cluster mesh, service outages or anomalies in one cluster can dynamically cause requests to fail over to other clusters, potentially in other regions or clouds.  This gives users the ability to run high-availability services in active-active configuration, optimizing compute utilization and traffic costs from a single control plane. Ambient Multi-Cluster will be available as an Alpha in Istio 1.27, which ships in August.
+
+### The future of extensibility
+
+The Istio project has offered several APIs for extensibility since launch, and none of them has been able to mature to Stable. Of those in use today, Envoy Filters are a powerful tool for tweaking internal proxy configuration, and modifying traffic flow, but are very difficult to use, and pose significant risk during upgrades, which can change the filter integrations in ways that cannot always be predicted.  WebAssembly emerged in 2019 as a powerful tool for Turing-complete modification of traffic, but community support for Wasm compilers and libraries outside the Istio ecosystem has waned substantially since that time, making it difficult for users to safely and securely use WebAssembly with Istio.
```

Comment:
```suggestion
The Istio project has offered several APIs for extensibility since launch, and none of them has been able to mature to Stable. Of those in use today, Envoy Filters are a powerful tool for tweaking internal proxy configuration, and modifying traffic flow, but are very difficult to use, and pose significant risk during upgrades, which can change the filter integrations in ways that cannot always be predicted.  WebAssembly (Wasm) emerged in 2019 as a powerful tool for Turing-complete modification of traffic, but community support for Wasm compilers and libraries outside the Istio ecosystem has waned substantially since that time, making it difficult for users to safely and securely use Wasm with Istio.
```